### PR TITLE
Fix bug with small default range in SpinBox

### PR DESCRIPTION
**Pull request by Czaki**
The default range in SpinBox is 0, 1000 which is really small. Try to create a widget with a value outside the default range causing the Exception. This PR adjusts the min-max range if the initial value is outside it.

### CODE REVIEW

**Comment by Czaki** [comment]
I'm not sure how to fix this error. The simplest method will be to hide arguments in kwargs and maybe overload?

**Comment by Czaki** [comment]
Thanks :)

**Review by tlambert03 — APPROVED** [review]
thank you @Czaki!

File: magicgui/widgets/_bases/ranged_widget.py [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
 
     Parameters
     ----------
     min : float, optional
-        The minimum allowable value, by default 0
+        The minimum allowable value, by default 0 (or `value` if `value` is less than 0)
     max : float, optional
-        The maximum allowable value, by default 1000
+        The maximum allowable value, by default 1000 (or `value` if `value` is greater
+        than 1000)
```
Comment (Czaki) [inline magicgui/widgets/_bases/ranged_widget.py:21]: ahh. I do not update this line. The default value is 999

**Pull request merged**